rlclientlib/utility/authorization.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cpprest/http_headers.h>
+
+using namespace web::http;
+
+namespace reinforcement_learning {
+  class api_status;
+  class i_authorization {
```

Comment:
why do we need this? Looks like there is no usage/children of this class. Is eventhub_http_authorization supposed to be inherited from here?

Reply:
Not anymore. I deleted the reference from the project file. Missed deleting the file. Deleted now.